### PR TITLE
Use mp4 in contain-inline-size-replaced.html

### PR DESCRIPTION
**Pull request by rwlbuis**
WebM is currently not supported in all cases in WebKit, especially
in the testing infrastructure. Since the video size rather than the
format is important for this test, use mp4 instead.

### CODE REVIEW

**Review by mrego — COMMENTED** [review]
I'd suggest to use `/media/white.webm` and `/media/white.mp4` or some other video already existent in `/media/` folder instead of adding a new video here.

File: css/css-contain/contain-inline-size-replaced.html [hunk 1]
```diff
@@ -28,10 +28,14 @@
       poster="support/blue-100x100.png"></video>
     <video class="inline-contained" data-expected-width="0" data-expected-height="100" poster="support/blue-100x100.png"
       controls></video>
-    <video class="inline-contained" data-expected-width="0" data-expected-height="240" src="support/white.webm"
-      controls></video>
-    <video class="inline-contained" data-expected-width="0" data-expected-height="240" src="support/white.webm"
-      controls></video>
+    <video class="inline-contained" data-expected-width="0" data-expected-height="240" controls>
+      <source src="support/white.webm" type=video/webm>
```
Comment (mrego) [inline css/css-contain/contain-inline-size-replaced.html:32]: We could do the same for this, but maybe in a follow up patch as there are more places that uses this.

**Review by mrego — APPROVED** [review]
LGTM.

File: css/css-contain/contain-inline-size-replaced.html (hunk 1 above)
Comment (rwlbuis) [reply css/css-contain/contain-inline-size-replaced.html:32]: Good idea, done.

**Pull request merged**